paper-input.html

```diff
@@ -67,6 +67,10 @@
         display: block;
       }
 
+      input {
```

Comment:
if we do
```css
paper-textarea {
  text-align: center;
}
```
we should get the label + textarea aligned on the center. Is this assumption right?